src/bin/pgcopydb/copydb.c

```diff
@@ -423,18 +423,15 @@ copydb_prepare_dump_paths(CopyFilePaths *cfPaths, DumpPaths *dumpPaths)
 	sformat(dumpPaths->extnspFilename, MAXPGPATH, "%s/%s",
 			cfPaths->schemadir, "extnamespaces.dump");
 
-	sformat(dumpPaths->preFilename, MAXPGPATH, "%s/%s",
-			cfPaths->schemadir, "pre.dump");
+	sformat(dumpPaths->dumpFilename, MAXPGPATH, "%s/%s",
+			cfPaths->schemadir, "pre_post.dump");
```

Comment:
The actual name from pg_dump and pg_restore documentations is “schema-only”. Let's use `schema.dump`.